arlas-hits-exporter/library.bash

```diff
@@ -0,0 +1,25 @@
+ArlasExporter::Docker_build () {
+  local DOCKER_DIRECTORY="$PROJECT_ROOT_DIRECTORY/arlas-hits-exporter/Docker"
+  local REPOSITORY_NAME="gisaia/arlas-hits-exporter"
+  local DOCKER_IMAGE_VERSION="$(cat "$PROJECT_ROOT_DIRECTORY/arlas-hits-exporter/docker_image_version")"
+
+  cp "$PROJECT_ROOT_DIRECTORY/common/Docker/configure_user_and_group.bash" "$DOCKER_DIRECTORY/"
+  cp "$PROJECT_ROOT_DIRECTORY/arlas-hits-exporter/export_data.bash" "$DOCKER_DIRECTORY/"
```

Comment:
If `export_data.bash`

- is only used in the Docker image
- is not shared with another Docker image

... then I think it should be placed in the `Docker/` directory, and this `cp` should be removed.